files/en-us/mdn/structures/macros/commonly-used_macros/index.html

```diff
@@ -236,7 +236,7 @@ <h3 id="Page_or_section_header_indicators">Page or section header indicators</h3
  <li><code><a href="https://github.com/mdn/yari/tree/master/kumascript/macros/SeeCompatTable.ejs">SeeCompatTable</a></code> should be used on pages that document <a href="/en-US/docs/MDN/Guidelines/Conventions_definitions#experimental">experimental features</a>. Example: <code>\{{SeeCompatTable}}</code> {{SeeCompatTable}}</li>
  <li><code><a href="https://github.com/mdn/yari/tree/master/kumascript/macros/deprecated_header.ejs">deprecated_header</a></code>: <code>\{{Deprecated_Header}}</code> {{Deprecated_Header}}</li>
 
- <li><code><a href="https://github.com/mdn/yari/tree/master/kumascript/macros/secureContext_header.ejs">secureContext_header</a></code>: <code>\{{SecureContext_Header}}</code> {{SecureContext_Header}}</li>
+ <li><code><a href="https://github.com/mdn/yari/tree/master/kumascript/macros/secureContext_header.ejs">secureContext_header</a></code>. Should be used on main pages like interface pages, API overview pages, and API entry points (e.g. <code>navigator.xyz</code>) but not necessarily on all sub-pages (not on all method and property pages). Example: <code>\{{SecureContext_Header}}</code> {{SecureContext_Header}}</li>
```

Comment:
I would be even more assertive. What about?
```suggestion
 <li><code><a href="https://github.com/mdn/yari/tree/master/kumascript/macros/secureContext_header.ejs">secureContext_header</a></code>. Should be used on main pages like interface pages, API overview pages, and API entry points (e.g. <code>navigator.xyz</code>) but usually not on sub-pages like method and property pages. Example: <code>\{{SecureContext_Header}}</code> {{SecureContext_Header}}</li>
```